sdk-scheme-adapter/v2_0_0/components/schemas/requestToPayStatus.yaml

```diff
@@ -0,0 +1,5 @@
+type: string
+enum:
+  - ERROR_OCCURRED
+  - WAITING_FOR_PARTY_ACCEPTANCE
+  - COMPLETED
```

Comment:
Is `COMPLETED` a good idea here?

Why not `PARTY_ACCEPTED`?

Reply:
I followed the existing transferStatus functionality. `COMPLETED` means the requestToPay operation is completed and the transactionRequestState can be anything like `RECEIVED`, `REJECTED`...etc

Incase of transfer, the SDK implementation is similar. The `COMPLETED` status means the operation is completed and the transfer status can be `ABORETED` `RESERVED` `COMMITTED`..etc

```
  - ERROR_OCCURRED
  - WAITING_FOR_PARTY_ACCEPTANCE
  - WAITING_FOR_QUOTE_ACCEPTANCE
  - COMPLETED
```

Reply:
Perfect.